File: Backend/JAVA/application/src/main/java/com/superhero/domain/model/Connections.java
package com.superhero.domain.model;

import java.io.Serializable;

import com.fasterxml.jackson.annotation.JsonProperty;

public final class Connections implements Serializable {

    @JsonProperty("group-affiliation")
    public String groupAffiliation;

    @JsonProperty("relatives")
    public String relatives;
    
    public Connections() {
    }

    public String getGroupAffiliation() {
        return groupAffiliation;
    }

    public void setGroupAffiliation(String groupAffiliation) {
        this.groupAffiliation = groupAffiliation;
    }

    public String getRelatives() {
        return relatives;
    }

    public void setRelatives(String relatives) {
        this.relatives = relatives;
    }
}

File: Backend/C#/SuperHero.Domain/Model/Connections.cs
using SuperHero.Domain.Model.BaseModel;
using System.Text.Json.Serialization;

namespace SuperHero.Domain.Model
{
    public class Connections
    {

        [JsonPropertyName("group-affiliation")]
        public string GroupAffiliation { get; set; }

        [JsonPropertyName("relatives")]
        public string Relatives { get; set; }
    }
}
